test/test-normandy-arguments.ts

```diff
@@ -1,26 +1,27 @@
 #!/usr/bin/env ts-node-script
-import { promises as fs } from "fs";
 
 import fetch from "node-fetch";
-import Ajv from "ajv";
 
-const NORMANDY_API_BASE = "https://normandy.cdn.mozilla.net/api";
-
-async function main() {
-  const ajv = new Ajv();
+import { typeGuards } from "..";
 
-  for await (const recipe of fetchEnabledRecipes()) {
-    const revision = recipe.approved_revision ?? recipe.latest_revision;
-    const typeName = convertActionNameToTypeName(revision.action.name);
-    const schema = await fs.readFile(`./schemas/normandy/${typeName}.json`);
+const NORMANDY_API_BASE = "https://normandy.cdn.mozilla.net/api";
 
-    if (!ajv.validate(schema, revision.arguments)) {
-      throw new Error(
-        `Recipe ${recipe.id} does not have valid arguments: ${ajv.errors}`
-      );
+describe("normandy schemas", () => {
+  it("all enabled Normandy recipes should match their schemas", async () => {
+    for await (const recipe of fetchEnabledRecipes()) {
+      const revision = recipe.approved_revision ?? recipe.latest_revision;
+      const typeName = convertActionNameToTypeName(revision.action.name);
+
+      // eslint-disable-next-line @typescript-eslint/ban-types
+      type asserterType = ((value: object) => void) | undefined;
+      const asserter = (typeGuards as Record<string, asserterType>)[`normandy_assert${typeName}`];
+      if (!asserter) {
+        throw new Error(`Asserter not found for action ${revision.action.name}`);
```

Comment:
Should we add a test case for this as well?
If so, do you want me to file a separate issue?

Reply:
I can't think of a reason this would happen, but if it did it would be a problem. If this happens there is probably a bug in AJV, and I'd like to know about it. I don't think we need a test for this. If this were Python, this would be `assert` statement to make sure that things are what we expect. JS doesn't have assert though, so this is the closest I could do.